fbchat/client.py

```diff
@@ -1230,28 +1240,28 @@ def setTypingStatus(self, status, thread_id=None, thread_type=None):
     END SEND METHODS
     """
 
-    def markAsDelivered(self, userID, threadID):
+    def markAsDelivered(self, threadID, messageID):
         """
         .. todo::
             Documenting this
         """
         data = {
-            "message_ids[0]": threadID,
-            "thread_ids[%s][0]" % userID: threadID
+            "message_ids[0]": messageID,
+            "thread_ids[%s][0]" % threadID: messageID
         }
 
         r = self._post(self.req_url.DELIVERED, data)
         return r.ok
 
-    def markAsRead(self, userID):
+    def markAsRead(self, threadID):
         """
         .. todo::
             Documenting this
         """
         data = {
+            "ids[%s]" % threadID: True,
```

Comment:
You said the issue was the capitilasation of the `True`, and I can verify that, but if that's the only issue, why not just write `'true'`?